### PR TITLE
ROC-8026: Inform defendant by letter that claimant was selected to use new features

### PR DESCRIPTION
**Pull request by AlokDatta**
Modified the defendant letter template to inform that - 'The claimant has been selected to use new features which apply to this claim.'

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-8026

### Change description ###
Modified the defendantPinLetter.html to introduce the expected text from the JIRA.

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProvider.java [hunk 1]
```diff
@@ -91,6 +93,7 @@ public DefendantPinLetterContentProvider(
             .getAddress()
         );
         content.put("hmctsEmail", staffEmailProperties.getRecipient());
+        content.put(NEW_FEATURES, claim.getFeatures() == null || claim.getFeatures().isEmpty() ? false : true);
```
Comment (andrewwa-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProvider.java:96]: Simplify
```suggestion
        content.put(NEW_FEATURES, claim.getFeatures() != null && !claim.getFeatures().isEmpty());
```

File: src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProviderTest.java [hunk 2]
```diff
@@ -140,4 +141,11 @@ public void shouldProvideHmctsEmail() {
         assertThat(content).containsEntry("hmctsEmail", STAFF_NOTIFICATIONS_RECIPIENT);
     }
 
+    @Test
+    public void shouldProvideHmctsNewFeatureConsent() {
+        Map<String, Object> content = provider.createContent(claim, DEFENDANT_PIN);
+
+        assertThat(content).containsEntry(NEW_FEATURES, Boolean.TRUE);
+    }
```
Comment (andrewwa-kainos) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProviderTest.java:149]: Also test the negative case please

File: src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProviderTest.java [hunk 3]
```diff
@@ -140,4 +141,11 @@ public void shouldProvideHmctsEmail() {
         assertThat(content).containsEntry("hmctsEmail", STAFF_NOTIFICATIONS_RECIPIENT);
     }
 
+    @Test
+    public void shouldProvideHmctsNewFeatureConsent() {
+        Map<String, Object> content = provider.createContent(claim, DEFENDANT_PIN);
+
+        assertThat(content).containsEntry(NEW_FEATURES, Boolean.TRUE);
```
Comment (pragnesh-patel) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProviderTest.java:148]: `assertTrue(content).containsEntry(NEW_FEATURES) ` ?

However Test will never fail once Feature is Permanent.  An additional Negative Test for absence of text, is maybe more helpful.

Comment (AlokDatta) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProviderTest.java:148]: I have added the negative test case. Thanks.

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProvider.java (hunk 1 above)
Comment (AlokDatta) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProvider.java:96]: Agreed. Suggestion taken.

File: src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProviderTest.java (hunk 2 above)
Comment (AlokDatta) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProviderTest.java:149]: Added the negative test case.
